artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis-service

```diff
@@ -61,7 +61,7 @@ status() {
     pid=`cat "${PID_FILE}"`
     # check to see if it's gone...
     ps -p ${pid} > /dev/null
-    if [ $? -eq 0 ] ; then
+    if [ $? -eq 0 ] && [ `pwdx ${pid} | awk '{print $2}'` == "$ARTEMIS_INSTANCE" ]; then
```

Comment:
I couldn't get this to work in my testing. I kept getting this:
```
./artemis-service: 64: [: /home/jbertram: unexpected operator
```
However, this worked:
```
if [ $? -eq 0 ] && [ "`pwdx ${pid} | awk '{print $2}'`" = "$ARTEMIS_INSTANCE" ] ; then
```